customers/schema/queries.py

```diff
@@ -202,56 +209,62 @@ class Query:
 
     @view_permission_required(CustomerProfile, BerthApplication, BerthLease)
     def resolve_berth_profiles(self, info, **kwargs):
-        berth_lease_customer = kwargs.pop("berth_lease_customer", False)
-        harbors = kwargs.pop("harbors", [])
-        piers = kwargs.pop("piers", [])
-        berths = kwargs.pop("berths", [])
-
-        marked_winter_storage_lease_customer = kwargs.pop(
-            "marked_winter_storage_lease_customer", False
-        )
-        marked_winter_storage_areas = kwargs.pop("marked_winter_storage_areas", [])
-        marked_winter_storage_places = kwargs.pop("marked_winter_storage_places", [])
-
-        unmarked_winter_storage_lease_customer = kwargs.pop(
-            "unmarked_winter_storage_lease_customer", False
-        )
-        unmarked_winter_storage_areas = kwargs.pop("unmarked_winter_storage_areas", [])
+        harbors = [h for h in kwargs.pop("harbors", []) if h is not None]
+        piers = [p for p in kwargs.pop("piers", []) if p is not None]
+        berths = [b for b in kwargs.pop("berths", []) if b is not None]
```

Comment:
Added a comment related to this on the other PR https://github.com/City-of-Helsinki/berth-reservations/pull/660#pullrequestreview-889545260

Reply:
Trivial / minor in my opinion. It would need some changes to UI also. Lets do it later if decided